providers/keycloak_oidc_test.go

```diff
@@ -0,0 +1,164 @@
+package providers
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"net/http/httptest"
+	"net/url"
+
+	. "github.com/onsi/gomega"
+
```

Comment:
I'm actually surprised out linter didn't yell at this for the import formatting, it's usually so particular. Can you ditch these extra newlines?